docs/configuration.md

```diff
@@ -73,6 +73,7 @@ If you want a more sophisticated rule, you can configure it in the config file:
 <?php
 
 $finder = new TwigCsFixer\File\Finder();
+$finder->in(__DIR__.'/templates'); // only lint files in the given directory
```

Comment:
```suggestion
$finder->in('templates');
```
Relatives files should be preferred to avoid issues like https://github.com/VincentLanglet/Twig-CS-Fixer/issues/237

Also, I'm not sure the comment is needed, the method call seems self-explanatory

Reply:
Well, I didn't know about Symfony Finder so I had to search a bit before I was able to set it up. The [in function](https://symfony.com/doc/current/components/finder.html#location) isn't that self explanatory in my opinion, that's why I added the comment.

Anyway, feel free to change whatever you want.